app/controllers/bigbluebutton_api_controller.rb

```diff
@@ -133,7 +133,7 @@ def get_meetings
 
   def get_meetings_disabled
     logger.debug('The get meetings api has been disabled')
-    render(xml: no_meetings_reponse)
+    render(xml: no_meetings_response)
```

Comment:
This appears to be a change for a different issue, and probably should have been in a separate PR.